core/state/state_object.go

```diff
@@ -412,12 +412,10 @@ func (s *StateObject) updateTrie(db Database) Trie {
 		s.originStorage[key] = value
 		var v []byte
 		if (value == common.Hash{}) {
-			s.setError(tr.TryDelete(key[:]))
 			s.db.StorageDeleted += 1
```

Comment:
`StorageDeleted & StorageUpdated` were added for storage metrics, but seems no longer used and can be deleted?

Reply:
duplicate update/delete  should cause by error merge since it has already removed by this pr
https://github.com/bnb-chain/bsc/pull/924/commits

Reply:
seems that some commit might not merged well.